Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -174,7 +162,7 @@ private let rootStackViewStyle: StackViewStyle = { (stackView: UIStackView) in
     |> \.axis .~ NSLayoutConstraint.Axis.horizontal
     |> \.translatesAutoresizingMaskIntoConstraints .~ false
     |> \.isLayoutMarginsRelativeArrangement .~ true
-    |> \.layoutMargins .~ UIEdgeInsets.init(topBottom: Styles.grid(5), leftRight: Styles.grid(2))
+    |> \.layoutMargins .~ UIEdgeInsets.init(topBottom: Styles.grid(5))
```

Comment:
We have both. I still need to pair with Danny to do a design sweep for the padding.

Reply:
Ok, so we might want to move that in a separate PR? I think it would be good to do it in the parent view and just use `UIStackView` `.setCustomSpacing(after:)` so that we don't run the risk of padding in subviews doubling up and looking weird or being tricky to maintain.

Reply:
Planning to fix all the spacing/padding for these views in a follow-up PR 👍